src/index.tsx

```diff
@@ -742,10 +742,10 @@ function getArrowPoints(element: ExcalidrawElement) {
 }
 
 function getDiamondPoints(element: ExcalidrawElement) {
-  const topX = Math.PI + element.width / 2;
-  const topY = element.height - element.height;
+  const topX = Math.floor(element.width / 2) + 1;
```

Comment:
Why the +1?

Reply:
And why the floor?

Reply:
Rough does not accept these numbers to be 0, I add 1 to avoid errors

Reply:
Can you add a comment saying that? Otherwise it’s an odd decision

Reply:
Sure!

Reply:
What is the error btw, that seems odd that rough would throw

Reply:
`InvalidStateError: Failed to execute 'createPattern' on 'CanvasRenderingContext2D': The image argument is a canvas element with a width or height of 0.`